docs/source/releasenotes.rst

```diff
@@ -13,6 +13,10 @@ Upcoming release
 - Library **PDF**: Keywords ``Template HTML To PDF`` and ``HTML To PDF`` will now
   create directory structure and overwrite existing file by default.
 
+- Library **RPA.Images**: Remove ``Save Format`` option from ``Take Screenshot`` 
```

Comment:
remove RPA prefix from the library name .... that applies to all libraries and is not needed in the release note

Reply:
Oh right whoops.

Reply:
Fixed